openedx/core/lib/xblock_builtin/xblock_discussion/xblock_discussion.py

```diff
@@ -134,3 +135,27 @@ def student_view_data(self):
         Returns a JSON representation of the student_view of this XBlock.
         """
         return {'topic_id': self.discussion_id}
+
+    @classmethod
+    def parse_xml(cls, node, runtime, keys, id_generator):
```

Comment:
Please add a docstring explaining that this method exist to preserve compatibility with the older discussion XModule OLX serialization format.